charmcraft.yaml

```diff
@@ -10,21 +10,17 @@ links:
   documentation: https://discourse.charmhub.io/t/identity-platform-login-ui-docs-index/12169
   source: https://github.com/canonical/identity-platform-login-ui-operator
   issues: https://github.com/canonical/identity-platform-login-ui-operator/issues
-
 assumes:
   - k8s-api
-
 containers:
   login-ui:
     resource: oci-image
-
 # This field populates the Resources tab on Charmhub.
 resources:
   oci-image:
     type: oci-image
     description: OCI image for login-ui
-    upstream-source: ghcr.io/canonical/identity-platform-login-ui:v0.20.0
-
+    upstream-source: ghcr.io/canonical/identity-platform-login-ui:main
```

Comment:
Is this intentional? Do we want to use `main` instead of a version tag? (cc @natalian98 @shipperizer)